content/integrations/docker.md

```diff
@@ -19,7 +19,7 @@ Get metrics from Docker in real time to:
 
 # Installation
 
-There are three ways to setup the Docker integration: install the agent on the host, on a single privileged container, and on each individual container.
+There are two ways to setup the Docker integration: install the agent on the host or on a single sidecar container.
```

Comment:
IIRC a sidecar container is tied to a "main" container, which is not the case here (dd-agent is running by itself).
I'd go with
> There are two ways to setup the Docker integration: running our docker-dd-agent container or installing the agent on the host.